common/config/rush/pnpm-lock.yaml

```diff
@@ -2549,44 +2586,44 @@ packages:
       - scheduler
     dev: false
 
-  /@fluentui/react-divider@9.2.82(@types/react-dom@18.3.1)(@types/react@18.3.12)(react-dom@18.3.1)(react@18.3.1):
-    resolution: {integrity: sha512-U3krzGswRxpFgMkXafkl12+R7R1SeX+JdTZpw8SMnvQsBroEWgy6hq8bWhh8AFUcqCoHIbieBOrRFV68pvIvGg==}
+  /@fluentui/react-divider@9.2.85(@types/react-dom@18.3.1)(@types/react@18.3.12)(react-dom@18.3.1)(react@18.3.1):
```

Comment:
This has _way_ more changes than just bumping the calling sdk package version. Revert the lockfile and just update the calling-sdk version. One handy way I do it is to manually edit the package.json files with the new calling sdk version (and the common-versions.json) then run rush update